lib/fedex/tracking_information.rb

```diff
@@ -44,6 +44,10 @@ def initialize(details = {})
         @delivery_at = Time.parse(details[:actual_delivery_timestamp])
       end
 
+      if details.has_key?(:estimated_delivery_timestamp)
+        @estimated_delivery_at = Time.parse(details[:estimated_delivery_timestamp])
```

Comment:
does `estimated_delivery_timestamp` include a time zone designation? It'll _probably_ parse it as UTC, which is _probably_ what you want, but just to make sure.